src/common/ChipCell/ChipCell.js

```diff
@@ -52,16 +52,16 @@ const ChipCell = ({
 
   const handleShowElements = useCallback(() => {
     if (!isEditMode || (isEditMode && visibleChipsMaxLength)) {
-      setShow(!show)
+      setShowHiddenChips(!showHiddenChips)
```

Comment:
please changed it to
```suggestion
      setShowHiddenChips(state => !state)
```